bitnami/cert-manager/Chart.yaml

```
@@ -35,4 +35,4 @@ maintainers:
 name: cert-manager
 sources:
 - https://github.com/bitnami/charts/tree/main/bitnami/cert-manager
-version: 1.1.1
+version: 1.1.2
```

Bumping version because 1.1.2 has already been released: https://github.com/bitnami/charts/blob/9bfb42efe9321d76528e0213d1210b3c09a85684/bitnami/cert-manager/Chart.yaml#L38

```suggestion
version: 1.1.3
```